src/locales/cs/translation.json

```
@@ -442,154 +442,154 @@
                 "link": "https://discord.com/invite/mX8yvYQTuq"
             },
             {
-                "title": "Official Guide: Organizing All Cards in Big Box",
+                "title": "Oficiální návod: Organizace všech karet v Big boxu",
                 "subtitle": "$t(communityLinks.contentType.video)",
-                "desc": "A more in-depth guide on how to fit all the cards inside the Big Box.",
+                "desc": "Hlubší návod na to, jak vměstnat všechny karty do Big boxu.",
                 "link": "https://youtu.be/Hj7oTXJOFfg"
             },
             {
-                "title": "Official FAQ: Clarifying ambiguous rules/situations",
+                "title": "Oficiální FAQ: Vyjasnění nejednoznačných pravidel/situací",
                 "subtitle": "$t(communityLinks.contentType.pdf)",
-                "desc": "Answers and clarification for potentially confusing or difficult situations.",
+                "desc": "Odpovědi a vyjasnění pro potenciálně matoucí či komplikované situace.",
                 "link": "https://www.dropbox.com/s/6fxx058x76kc9rn/FAQ_ENG_v01.pdf?dl=0"
             },
             {
-                "title": "BoardGameGeek Old World Files",
+                "title": "Soubory pro Starý Svět na BoardGameGeeku",
                 "subtitle": "$t(communityLinks.contentType.link)",
-                "desc": "Another community where members upload their own creations, automata, guides, and more.",
+                "desc": "Druhá komunita, kde uživatelé nahrávají své vlastní výtvory, automatické protihráče, návody a další.",
                 "link": "https://boardgamegeek.com/boardgame/331106/witcher-old-world/files"
             },
             {
-                "title": "Witcher Monster Tray Position",
+                "title": "Umístění jednotlivých nestvůr v insertech",
                 "subtitle": "$t(communityLinks.contentType.member)",
-                "desc": "A detailed guide on locations of all the monster minis in their respective trays (Big Box only).",
+                "desc": "Detailní návod na dohledání všech monster na jejich místech v insertech (pouze pro Big box).",
                 "link": "https://boardgamegeek.com/filepage/259976/witcher-monster-tray-position"
             },
             {
-                "title": "Card and Token Organizers",
+                "title": "Organizéry karet a tokenů",
                 "subtitle": "$t(communityLinks.contentType.member)",
-                "desc": "3D printer files for card organizers to better fit in the Big Box, as well as token organizers.",
+                "desc": "Soubory pro 3D tisk pro organizéry na karty nebo tokeny pro lepší umístění v Big boxu.",
                 "link": "https://thingiverse.com/thing:6078120"
             }
         ]
     },
-    "lostMount": {
-        "title": "Lost Mount Locations"
+    "Mount": {
```

This needs to stay as "lostMount" to not break references to this title used elsewhere.

```suggestion
    "lostMount": {
```

---

I don't even know how this happened :-D, thanks for checking and fixing